src/Jackett.Common/Indexers/BJShare.cs

```diff
@@ -394,6 +394,9 @@ private async Task<List<ReleaseInfo>> ParseUserSearchAsync(TorznabQuery query)
                                              titleElements[2] + " " + titleElements[4] + " " + string.Join(
                                                  " ", titleElements.Skip(6));
 
+                        if (Regex.IsMatch(release.Description, "(Dual|[Nn]acional|[Dd]ublado)"))
```

Comment:
You would probably need to add it on ParseLast24HoursAsync method aswell, that is run when rss sync is called

Reply:
Ok, I will check it. tks.

Reply:
Done.